### ISSUE TITLE
About generated images

### ISSUE THREAD

**Issue by nhk19980516** [eladrich/pixel2style2pixel#54]
I fine-tune a Stylegan model based a Disney datasets, and the model can generate good result. But I used the model to be the decoder of the PSP model during my training process, the generated result is worst. I appreciate it if somebody can help me.
![246000](https://user-images.githubusercontent.com/52226113/101459831-b2e05280-3973-11eb-9501-644e6b5bd4e0.jpg)

**Comment by yuval-alaluf** [comment]
Since our toonify model is trained on a different StyleGAN generator, the loss lambdas we use may not be optimal for your training. I would consider playing with the loss lambdas to find something that works best for you.  

In any case, even though our toonify model worked well, training with no paired data (i.e. only real images) is quite challenging. If you have an option to generate paired data this may help in training pSp with your StyleGAN model. You can check out the following Google Colab from Justin Pinkney to see an example of how to generate the paired images:
https://colab.research.google.com/drive/1s2XPNMwf6HDhrJ1FMwlW1jl-eQ2-_tlk?usp=sharing#scrollTo=cuMEHnpmI1Mj

**Comment by johndpope** [comment]
Does the stylegan2-ada make the colab document / git repo https://github.com/justinpinkney/stylegan2 out of date?

**Comment by yuval-alaluf** [comment]
> Does the stylegan2-ada make the colab document / git repo https://github.com/justinpinkney/stylegan2 out of date?

Not at all. StyleGAN-ADA has the same architecture as StyleGAN2 so any model trained with the new ADA method will work with pSp. Please note that we do not use the StyleGAN2 code from the repository you linked. We use rosinality's Pytorch implementation which can be found here: 
https://github.com/rosinality/stylegan2-pytorch

Given a StyleGAN tensorflow model trained using ADA, you can use rosinality's conversion script to convert the model to Pytorch and then load the generator as you would with any StyleGAN2 generator.

**Comment by johndpope** [comment]
thanks for extra info. I'm looking down the barrel of upgrading nvidia stylegan2 code to support tensorflow2 just so I can get my 3090 card to work properly.  switching over to pytorch not out of the question.
